com.hamrick.VueScan.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.hamrick.VueScan</id>
+	<name>VueScan</name>
+	<project_license>LicenseRef-proprietary</project_license>
+	<developer_name>Hamrick Software</developer_name>
+	<summary>Scanning software for 5000+ scanners</summary>
+	<metadata_license>CC0-1.0</metadata_license>
+	
+	<description>
+		<p>VueScan is a scanning program that is compatible with over 5680 scanners - and is available for Windows, Mac OS X and Linux.</p>
```

Comment:
I'd like it to be clearer in the description that the version you install out of the box is a free trial and needs payment to unlock.